### PR TITLE
Batch of notice fixes

### PR DESCRIPTION
**Pull request by MrPetovan**
Address:
- https://github.com/friendica/friendica/issues/9252#issuecomment-740069639
- https://github.com/friendica/friendica/issues/9252#issuecomment-740052103
- https://github.com/friendica/friendica/issues/9252#issuecomment-739534960

### CODE REVIEW

File: src/Module/Admin/Item/Source.php [hunk 1]
```diff
@@ -33,7 +33,7 @@ public static function content(array $parameters = [])
 	{
 		parent::content($parameters);
 
-		$guid = basename($_REQUEST['guid'] ?? '') ?: $parameters['guid'];
+		$guid = basename($_REQUEST['guid'] ?? $parameters['guid'] ?? '');
```
Comment (annando) [inline src/Module/Admin/Item/Source.php:36]: AFAIK there is had been something with these nested things in PHP8.

Comment (MrPetovan) [reply src/Module/Admin/Item/Source.php:36]: Only with the ternary operator Like this : `a ? b : c ? d : e`. There is no ambiguity with binary operators like `?:` and `??`.

Comment (annando) [reply src/Module/Admin/Item/Source.php:36]: Okay.

Comment (MrPetovan) [reply src/Module/Admin/Item/Source.php:36]: And you went to bed?

Comment (annando) [reply src/Module/Admin/Item/Source.php:36]: I wanted to have a look at it after work - but I came home really late, so I forgot about it.

**Pull request merged**